content/posts/2023/reana-0.9.1.md

```diff
@@ -0,0 +1,109 @@
+REANA 0.9.1 also introduces the new `prune` command, which can be used to clean up your workflows by deleting all the files in their workspaces, with the exclusion of input and output files, so that you don't have to delete each file one by one.
+
+```console
+$ reana-client prune -w reana-demo-root6-roofit
+==> SUCCESS: The workspace has been correctly pruned.
+```
+
+Take a look to the related [Prune the workspace](https://docs.reana.io/advanced-usage/user-quotas/#prune-the-workspace) documentation page to learn more about this new command.
+
+### ... and much more!
+
+This new release brings many more small improvements. For example, it's now possible to stop workflows from the web UI, by clicking on the _Stop workflow_ button in the actions menu.
+Furthermore, restarted Snakemake workflows now report the correct total number of jobs, by excluding cached jobs that were simply reused from previous runs in the workspace and not really executed.
+
+## What's new for the administrators?
+
+### Keycloak
+
+REANA provides users with two ways to log in, either using username/password or via an external SSO provider.
+Up until now support for external SSO systems was limited to CERN SSO, but REANA 0.9.1 removes this limitation by being able to integrate with any custom Keycloak instance.
+
+If you are interested in using Keycloak together with REANA, please take a look to the related [Keycloak Single Sign-On configuration](https://docs.reana.io/administration/configuration/configuring-access/#keycloak-single-sign-on-configuration) documentation page.
+
+### Launcher examples
+
+REANA 0.9.0 introduced the possibility to launch workflows from external sources like GitHub, GitLab or Zenodo.
+It is now possible to customise the demo examples that are showcased in the Launch-on-REANA welcome page via the new `components.reana_ui.launcher_examples` Helm value.
+In this way you can expose your own demos to your users, instead of the default ones provided by REANA.
+
+![Launcher examples](/images/reana-0.9.1-launcher-examples.png)
+
+<!-- TODO: add link to docs/blog post here -->
+
+### Automatic closure of inactive sessions
+
+It might happen that users open some interactive sessions and forget to close them when they are not needed anymore.
+To keep resource utilisation under control, REANA 0.9.1 allows to set a maximum time period of inactivity after which inactive sessions will be closed.
+The inactivity period can be customised with the `interactive_sessions.maximum_inactivity_period` Helm value.
+
+To learn more about this feature, please have a look at the related [Configuring interactive sessions](https://docs.reana.io/administration/configuration/configuring-interactive-sessions/#auto-closure-of-inactive-sessions) documentation page.
+
+### Performance and stability improvements
+
+There are many improvements in performance and stability that have landed in this new REANA release:
```

Comment:
There are many improvements in stability and performance of the REANA cluster that are coming with the 0.9.1 release update: